data/journal_check/bug_refs.json

```diff
@@ -986,5 +986,35 @@
             "sle": ["16.0"]
         },
         "type": "ignore"
+    },
+    "vmware-firmware-bug": {
+        "description": "[Firmware Bug]:.*APIC ID mismatch",
+        "products": {
+            "opensuse": ["Tumbleweed"],
+            "microos":  ["Tumbleweed"],
+            "sle-micro": ["6.0","6.1","6.2"],
+            "sle": ["16.0"]
+        },
+        "type": "ignore"
+    },
+    "vmware-io-error": {
```

Comment:
This should be the same as https://bugzilla.suse.com/show_bug.cgi?id=1228559 aka L#766. I guess it is basically no floppy device present message

Reply:
How? AFAICS those are two different error messages in the journal that need to be treated as two individual objects (even if they originate from a similar source).

Reply:
Since they are to be ignored, and we already do that on L#766 I assume that `"vmware-io-error"` wont be used anymore and it is just redundant.

Reply:
Let's try! https://openqa.suse.de/tests/19181724